### PR TITLE
Better test repo

### PR DESCRIPTION
**Pull request by garemoko**
Re-make of https://github.com/LearningLocker/Moodle-Log-Expander/pull/36

Ready for review and merge.

### CODE REVIEW

File: tests/TestRepository.php [hunk 1]
```diff
@@ -9,56 +9,22 @@ class TestRepository extends MoodleRepository {
...
-            'value' =>  '2',
-            'timemodified' => 1433946702,
-        ];
+    protected function readStoreRecord($type, array $query, $index = 0) {
+
+        $file = file_get_contents(__DIR__ ."/fakeDB.json");
 
-        if ($type == 'question_attempt_steps') {
-            $response['state'] = 'gradedright';
+        $fakeMoodleDatabase = json_decode($file, true);
```
Comment (ryansmith94) [inline tests/TestRepository.php]: Should probably be loaded at the top of the file instead or in the constructor to avoid repeatedly reading the file.

**Comment by ryansmith94** [comment]
Just one comment here @jlowe64.

**Comment by garemoko** [comment]
@ryansmith94 done

**Comment by ryansmith94** [comment]
Awesome! Cheers @garemoko! 👍

**Comment by ryansmith94** [comment]
Ready to merge @jlowe64 :smile:

**Comment by jlowe64** [comment]
@garemoko you are amazing!

**Pull request merged**